openwisp_controller/config/apps.py

```diff
@@ -161,3 +162,12 @@ def enable_cache_invalidation(self):
             DeviceChecksumView.invalidate_checksum_cache,
             dispatch_uid='invalidate_checksum_cache',
         )
+
+    def notification_cache_update(self):
+        from openwisp_notifications.handlers import register_notification_cache_update
+
+        register_notification_cache_update(
+            self.device_model,
+            device_name_changed,
+            signal_uid="Config_Device_notification_cache_invalidation",
```

Comment:
```suggestion
            signal_uid='config_device_notification_cache_invalidation',
```

I don't know whether there is a convention to keep `dispatch_uid` in lowercase, but let's keep it lowercase for consistency with rest of OpenWISP's code.

Reply:
Hold on from making this change. We might need to update it again because of https://github.com/openwisp/openwisp-notifications/pull/168#discussion_r596918425

Reply:
I feel really sorry that you are waiting for me to make change. Actually, I am not well right now so I am able to work with my full effort. Can we please wait for 1 or 2 days for it?

Reply:
No Problem @niteshsinha17 I will wait for making further changes.

Reply:
Thanks @codesankalp